lib/workers/global/initialize.ts

```diff
@@ -68,13 +68,13 @@ export async function globalInitialize(
   config_: AllConfig
 ): Promise<RenovateConfig> {
   let config = config_;
+  setGlobalHostRules(config);
   await checkVersions();
```

Comment:
I think we should flip this lines. we don't need host rules before system version check